src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -201,7 +201,14 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
     const {data, selectedItem, showIcon, instructionLabel, timeLabel, player} = this.props;
     const {id, previewImage, itemType, displayTime, liveCuePoint, groupData, displayTitle, displayDescription, startTime} = data;
     const {imageLoaded} = this.state;
-    const ariaLabelTitle: string = (typeof displayTitle === 'string' && displayTitle ? displayTitle : displayDescription) || '';
+    const ariaLabelTitle =
+      typeof data.displayTitle === 'string'
```

Comment:
please rewrite this condition in a more readable way, like using ifs

also, do we need to check the type of ariaLabel as well ?
in that case we could just create a function like

getValue([data.displayTitle, data.ariaLabel, data.displayDescription])
function getValue(arr)
    for a of arr
      if a && typeof a === "string" return a
return "";